.github/workflows/node.js.yml

```diff
@@ -20,9 +20,9 @@ jobs:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
```

Comment:
Maybe also worth it to add caching?
https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data